services/web-server/src/data/AuthorizationCode.js

```diff
@@ -5,7 +5,7 @@ const AuthorizationCode = Entity.configure({
   version: 1,
   partitionKey: Entity.keys.ConstantKey('authorizationCodes'),
   rowKey: Entity.keys.StringKey('code'),
-  signEntities: true,
+  signEntities: false,
```

Comment:
This will create a conflict with the existing web-server migration PR. If changing this is not necessary for this service since we don't care about old data then maybe we can avoid changing it here.

Reply:
Good spot, thank you -- I meant to remove changes for web-server.  So many things going on :(